oe_theme.theme

```diff
@@ -975,6 +975,10 @@ function oe_theme_preprocess_paragraph__oe_text_feature_media(array &$variables)
     // The media entity is not available anymore, bail out.
     return;
   }
+
+  // Retrieve the correct media translation.
+  $media = \Drupal::service('entity.repository')->getTranslationFromContext($media, $paragraph->language()->getId());
```

Comment:
We don't need this right, as it will get it from the context?
```
$paragraph->language()->getId()
```

Reply:
If I remove these, tests are failing as it loads the English version of the file.